src/autoscaler/db/sqldb/scalingengine_sqldb.go

```diff
@@ -60,17 +62,24 @@ func (sdb *ScalingEngineSQLDB) SaveScalingHistory(history *models.AppScalingHist
 	return err
 }
 
-func (sdb *ScalingEngineSQLDB) RetrieveScalingHistories(appId string, start int64, end int64, orderType db.OrderType) ([]*models.AppScalingHistory, error) {
+func (sdb *ScalingEngineSQLDB) RetrieveScalingHistories(appId string, start int64, end int64, orderType db.OrderType, includeAll bool) ([]*models.AppScalingHistory, error) {
 	var orderStr string
 	if orderType == db.DESC {
 		orderStr = db.DESCSTR
 	} else {
 		orderStr = db.ASCSTR
 	}
+	var includeStr string
+	if !includeAll {
+		includeStr = fmt.Sprintf(" AND status != %d", models.ScalingStatusIgnored)
```

Comment:
It is not a good way to use "!=" in a sql query as it will scan all the records. It is better to use "in()" or "=".

Reply:
in() will still have the issue, so I moved the filtering out of db query